src/applications/mhv/medical-records/util/helpers.js

```diff
@@ -85,3 +86,15 @@ export const processList = list => {
   if (list?.length === 1) return list.toString();
   return emptyField;
 };
+
+/**
+ * @param {Error} error javascript error
+ * @param {String} page name of the page sending the error
+ * @returns {undefined}
+ */
+export const sendErrorToSentry = (error, page) => {
+  Sentry.captureException(error);
+  Sentry.captureMessage(
+    `MHV - Medical Records - ${page} - PDF generation error`,
```

Comment:
Couple of thoughts - could this be moved to `shared/util` so that other MHV apps can also use it? Second, could the "PDF generation error" text be removed or parameterized so that this can be used anywhere? I'm ok with changes being done in a follow-on issue or tech debt ticket too 🙂